### PR TITLE
Public OSS demo 3 blog post

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Public OSS demo 3 blog post

### CODE REVIEW

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "PrestaShop Public OSS Demo - March 2021 session"
+subtitle: "We held the third session of the open source project's public demo"
+date:   2021-04-08 16:00:00
```
Comment (matks) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]:
```suggestion
date:   2021-04-09 12:00:00
```

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+layout: post
+title:  "PrestaShop Public OSS Demo - March 2021 session"
+subtitle: "We held the third session of the open source project's public demo"
+date:   2021-04-09 12:00:00
+authors: [ MathieuFerment ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+On the 31st of March, the maintainers team held again a video conferencing for the project public demonstrations session, streamed live on [YouTube](https://youtu.be/UJ_XQs8trs4).
```
Comment (eternoendless) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]:
```suggestion
The project's third monthly public demo session was held and [streamed live](https://youtu.be/UJ_XQs8trs4) on March 31, 2021.
```

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## The third Public open source demonstrations agenda
+
```
Comment (eternoendless) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]:
```suggestion
```

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+The session agenda was as follows:
```
Comment (eternoendless) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]:
```suggestion
The session's agenda was as follows:
```

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- [01:02:30](https://youtu.be/UJ_XQs8trs4?t=3750) Closure
+
+You can use the timestamps to jump directly to the parts you want to watch.
+
+[Watch the replay on YouTube](https://youtu.be/UJ_XQs8trs4){: .btn .btn-lg .btn-info}
+{: .text-center}
+
+## Next session in April
+
+The next session of public PrestaShop open source demonstrations will be streamed on [**Wednesday 28st of April 2021, 2pm CET**](https://www.youtube.com/watch?v=VKb39T_6WHU).
```
Comment (eternoendless) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]: Don't forget that timezone has changed to summer time!

```suggestion
The next session will be streamed live on [**Wednesday, April 28, 2021 at 2pm CEST**](https://www.youtube.com/watch?v=VKb39T_6WHU).
```

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+We expect this session to be spectacular as the Feature Freeze of PrestaShop 1.7.8 should be done by this time, meaning many projects will be complete.
```
Comment (eternoendless) [inline news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]: Spectacular miiiiiight be a little bit of an overstatement, don't you think?
```suggestion
We expect this session to be particularly interesting as PrestaShop 1.7.8 should have entered Feature Freeze by this time, meaning many projects will be complete.
```

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md (hunk 3 above)
Comment (eternoendless) [reply news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]: Do we need this title? I find it kind of repetitive

Comment (matks) [reply news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]: I added it on purpose to make it repetitive and keep a common format with all "blog posts about a demo that happened".

We can remove it to lighten it :)

Comment (matks) [reply news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md]: Removed https://github.com/PrestaShop/prestashop.github.io/pull/878/commits/4938ac067e20450fd468403d238490b93be62a53

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matks

**Pull request merged**